package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tidepool/viz",
-  "version": "0.8.8",
+  "version": "0.8.9",
```

Comment:
as this is new functionality should really be a `minor` bump i  feel `0.9.0` especially as we are so adverse to calling something a v1  :)

Reply:
:) Good Point